modules/search/client/controllers/search.client.controller.js

```diff
@@ -41,6 +41,11 @@ function SearchController(
   // Visibility toggle for search place input on small screens
   vm.isPlaceSearchVisible = false;
 
+  if (offer && offer.user) {
```

Comment:
You can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) in case like this:

`if (offer?.user?.seen)`